backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1650,3 +1650,17 @@ databaseChangeLog:
                   defaultValueNumeric: 200
                   constraints:
                     nullable: false
+  - changeSet:
+      id: add-test-length
+      author: emma@skylight.digital
+      comment: Add test length column to the device type table. Test length will be based on the model used.
+      changes:
+        - addColumn:
+            tableName: device_type
+            columns:
+              - column:
+                  name: test_length
+                  type: int
+                  remarks: The length (in minutes) that the test will take.
+                  valueNumeric: 15 # backdate
```

Comment:
I think I'm responsible for this malapropism making it into the changelog, so I should go back and fix the original comment, but "backdate" is clearly the wrong word—it should probably be "back-propagate" or conceivably something more than one explanatory word. 😁

Reply:
Haha yeah, that was a leftover from some copy-pasting 😁  Updated now, hopefully it's clearer.